### ISSUE TITLE
Update to Angular RC5

### ISSUE THREAD

**Issue by charsleysa** [ngrx/router#166]
Hi there,

I see there is a feature branch for upgrading to Angular RC5.

Is there something that needs doing? Or is it just waiting to be merged into master and packaged onto NPM?

Would be keen to help out if something needs to be done as I'm wanting to upgrade our project to move away from the deprecated API's that will be removed in the next Angular release.

Cheers!

**Comment by brandonroberts** [comment]
The ngrx/router project is deprecated in favor of the [Angular 2 Router](https://angular.io/docs/ts/latest/guide/router.html) which shares a similar API and includes all the same features. I've added a deprecation warning to the README

**Comment by fxck** [comment]
@MikeRyan52 @brandonroberts this is just lazy me, that doesn't really want to migrate, speaking, but wouldn't it be worth making @ngrx/router work with rc.5, which would(or should) mean that it should be compatible with the final release and THEN deprecate it?

**Comment by littleStudent** [comment]
@fxck brandon pointed out to me yesterday on gitter, that its should be compativle with rc5, but will break with rc6

**Comment by brandonroberts** [comment]
@fxck I have a plunker here that shows it working with RC5. All the deprecations will be removed once RC6 lands, and the router uses the deprecated `ComponentResolver` which is will why it will break then. Even if we updated it to work with RC6, it still needs an overhaul to be able to support lazy loading of modules instead of components.

http://plnkr.co/edit/jz8Fn8?p=preview

**Comment by fxck** [comment]
I see. https://github.com/ngrx/router-store/issues/3 this would be nice then.
